circle/circle.go

```diff
@@ -20,4 +21,10 @@ func main() {
 	// ++ Your code here! ++
 	//
 	// Hint: Use math.Pi constant (https://pkg.go.dev/math#pkg-constants) to obtain π value.
-}
\ No newline at end of file
+	var r float64
+	const π = math.Pi
+	fmt.Println("Enter the radius:")
+	fmt.Scan(&r)
+	fmt.Println("Perimeter:", 2*π*r)
```

Comment:
Consider introducing a separate variable for this expression. Same for `π*(r*r)` in this file and for other expressions in other programs.